src/Nanoid/Nanoid.cs

```diff
@@ -142,10 +142,19 @@ public static class SubAlphabets
         /// </summary>
         public const int DefaultIdSize = 21;
 
+        /// <summary>
+        /// Backing field for <see cref="Nanoid.GlobalRandom"/>.
+        /// </summary>
+        [ThreadStatic]
+        private static CryptoRandom _globalRandom;
+
         /// <summary>
         /// Global <see cref="CryptoRandom"/> instance used to conveniently generate Nanoids.
         /// </summary>
-        private static readonly CryptoRandom GlobalRandom = new CryptoRandom();
+        /// <remarks>
+        /// Lazily initialized in order to account for the ThreadStatic attribute (learn more here: https://stackoverflow.com/a/18086509)
+        /// </remarks>
+        public static CryptoRandom GlobalRandom => _globalRandom ?? (_globalRandom = new CryptoRandom());

```

Comment:
Initializing `_globalRandom` directly in my understanding would not behave as expected when using `[ThreadStatic]` as it uses the static constructor of the class which is only called once in the lifetime of the entire program, whereas what we want here is to initialize the variable per thread when it is used. Thus the recommended solution based on my reading is to have a getter such as this one that lazily initializes it upon usage.

I also made it public in case there are people who wish to use this but need to preload it. I haven't personally measured the initialization of `CryptoRandom` but I think this enables people who think it might be an issue to deal with it without requiring more changes to the library.